src/templates/renew-membership-page.js

```diff
@@ -0,0 +1,129 @@
+import React, { useState, useEffect } from "react"
+import { connect } from 'react-redux'
+import { navigate } from "gatsby";
+import Layout from '../components/Layout'
+import TopBar from "../components/TopBar";
+import NavbarV2 from "../components/NavbarV2";
+import Header from "../components/Header";
+import SEO from "../components/SEO";
+import 'openstack-uicore-foundation/lib/css/components/index.css';
+import { AjaxLoader } from "openstack-uicore-foundation/lib/components";
+import { renewMembership } from "../actions/user-actions";
+
+import leftArrow from '../img/svg/arrow-left.svg'
+
+import '../style/modules/_renew-membership.scss';
+import LinkComponent from "../components/LinkComponent";
+
+export const RenewMembershipPageTemplate = ({
+  isLoggedUser,
+  location,
+  loading,
+  renewMembership,
+  userProfile
+}) => {
+
+  const [acceptedTerms, setAcceptedTerms] = useState(false);
+
+  useEffect(() => {
+    // if (userProfile.membership_type === "Individual") navigate("/a/profile")
```

Comment:
why is this comment out ? review https://github.com/OpenStackweb/osf-website/pull/669/files#r2180824251